searchcore/src/vespa/searchcore/proton/flushengine/flushengine.cpp

```diff
@@ -100,10 +102,10 @@ FlushEngine &
 FlushEngine::close()
 {
     {
-        MonitorGuard strategyGuard(_strategyMonitor);
-        MonitorGuard guard(_monitor);
+        std::lock_guard<std::mutex> strategyGuard(_strategyLock);
+        std::lock_guard<std::mutex> guard(_lock);
```

Comment:
I prefer a using statement for std::lock_guard<std::mutex>. That helps readability significantly.